elixir/apps/web/lib/web/live/settings/api_clients/index.ex

```diff
@@ -0,0 +1,112 @@
+defmodule Web.Settings.ApiClients.Index do
+  use Web, :live_view
+  alias Domain.Actors
+
+  def mount(_params, _session, socket) do
+    unless Domain.Config.global_feature_enabled?(:api_client_ui),
+      do: raise(Web.LiveErrors.NotFoundError)
+
+    socket =
+      socket
+      |> assign(page_title: "API Clients")
+      |> assign_live_table("actors",
+        query_module: Actors.Actor.Query,
+        sortable_fields: [
+          {:actors, :name},
+          {:actors, :status}
+        ],
+        hide_filters: [
+          :provider_id,
+          :type
+        ],
+        callback: &handle_api_clients_update!/2
+      )
+
+    {:ok, socket}
+  end
+
+  def handle_params(params, uri, socket) do
+    socket = handle_live_tables_params(socket, params, uri)
+    {:noreply, socket}
+  end
+
+  def handle_api_clients_update!(socket, list_opts) do
+    with {:ok, actors, actors_metadata} <-
+           Actors.list_actors_by_type(socket.assigns.subject, :api_client, list_opts) do
+      socket =
+        assign(socket,
+          actors: actors,
+          actors_metadata: actors_metadata,
+          page_title: "API Clients"
```

Comment:
This assign is not needed here

Reply:
Whoops, good catch!  This has been updated.